src/app.ts

```diff
@@ -20,12 +21,13 @@ const main = async () => {
 
   await dbConnection();
 
-  var bodyParser = require('body-parser');
+  // eslint-disable-next-line vars-on-top, global-require, @typescript-eslint/no-var-requires
```

Comment:
what is this linter disable doing?

Reply:
Will remove the disable and also bodyParser since it is deprecated now